sdk/src/main/java/io/dapr/runtime/Dapr.java

```diff
@@ -58,6 +60,14 @@ public static DaprRuntime getInstance() {
     return instance;
   }
 
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public String serializeSubscribedTopicList() throws IOException {
+    return new String(this.serializer.serialize(this.getSubscribedTopics()), StandardCharsets.UTF_8);

```

Comment:
Is the last arg needed?  That is the java default.

Reply:
It is not needed, but it is good to specify which Charset you need, otherwise (from Java docs):

"Constructs a new String by decoding the specified array of bytes using the platform's default charset."